contracts/src/types/RequestLock.sol

```diff
@@ -49,6 +49,13 @@ struct RequestLock {
     /// based on request digest instead of index. As a friction, this would introduce a second
     /// user-facing concept of what identifies a request.
     bytes8 fingerprint;
+    ///
+    /// Storage slot 3
+    ///
+    /// @notice The address of the contract to call when the proof is delivered. Address is 0 if no callback is required.
+    address callbackAddress;
+    /// @notice If a callback is requested, the maximum gas to use for the callback.
+    uint96 callbackGasLimit;
```

Comment:
We could put this into the assessor journal instead, to save on storage costs.

Reply:
Thats a great idea